src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -3071,4 +3151,46 @@ protected function isProtectedPath($path)
 
 		return true;
 	}
+
+
+	/**
+	 * Set the picker value
+	 */
+	protected function setPickerValue()
+	{
+		$varValue = \Input::get('value');
+
+		if (empty($varValue))
+		{
+			return;
+		}
+
+		$varValue = array_filter(explode(',', $varValue));
+
+		if (empty($varValue))
+		{
+			return;
+		}
+
+		$this->arrPickerValue = $varValue;
+
+		// TinyMCE will pass the path instead of the ID
+		if (strpos($varValue[0], \Config::get('uploadPath') . '/') === 0)
```

Comment:
Again, this is not new code. We are using this already:

https://github.com/contao/core-bundle/blob/master/src/Resources/contao/widgets/FileSelector.php#L625